src/components/views/messages/MessageActionBar.tsx

```diff
@@ -452,7 +451,9 @@ export default class MessageActionBar extends React.PureComponent<IMessageAction
 
     public render(): JSX.Element {
         const toolbarOpts = [];
-        if (canEditContent(this.props.mxEvent)) {
+        const isFavouritePageCheck = window.location.href.split("/").pop() === "favourite_messages";
```

Comment:
There ought to be a better way to do this...

Reply:
Now, I am back.

Reply:
Thanks, but i couldn't figure out a better way yet. your suggestion is welcomed